Using TensorFlow backend.
Saved model found
Traceback (most recent call last):
  File "train_binary.py", line 45, in <module>
    classifier.create_model()
  File "/home/neeraj/Desktop/resnet-tf/src/model/cnn.py", line 115, in create_model
    self.model = load_model(self.weights_file)
  File "/home/neeraj/miniconda3/envs/dl/lib/python3.6/site-packages/keras/engine/saving.py", line 417, in load_model
    f = h5dict(filepath, 'r')
  File "/home/neeraj/miniconda3/envs/dl/lib/python3.6/site-packages/keras/utils/io_utils.py", line 186, in __init__
    self.data = h5py.File(path, mode=mode)
  File "/home/neeraj/miniconda3/envs/dl/lib/python3.6/site-packages/h5py/_hl/files.py", line 312, in __init__
    fid = make_fid(name, mode, userblock_size, fapl, swmr=swmr)
  File "/home/neeraj/miniconda3/envs/dl/lib/python3.6/site-packages/h5py/_hl/files.py", line 142, in make_fid
    fid = h5f.open(name, flags, fapl=fapl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5f.pyx", line 78, in h5py.h5f.open
OSError: Unable to open file (file signature not found)
